HYCOM-interface/CMakeLists.txt

```diff
@@ -24,7 +28,11 @@ if(CMAKE_C_COMPILER_ID MATCHES "GNU")
   set(CMAKE_C_FLAGS_DEBUG "-O0")
   set(CMAKE_C_LINK_FLAGS "")
 elseif(CMAKE_C_COMPILER_ID MATCHES "Intel")
-  set(CMAKE_C_FLAGS "-g -traceback -xSSE4.2 -mcmodel=small")
+  if(SUPAFAST)
+    set(CMAKE_C_FLAGS "-g -traceback -mcmodel=small")
+  else()
+    set(CMAKE_C_FLAGS "-g -traceback -xSSE4.2 -mcmodel=small")
+  endif()
```

Comment:
Same here, please make this change (removing -xSSE4.2) when using Intel Compiler. Thanks!